packages/bot-web-ui/src/components/bot-notification-messages/bot-notification-messages.tsx

```diff
@@ -5,20 +5,30 @@ import { connect } from 'Stores/connect';
 
 interface TBotNotificationMessagesProps {
     is_drawer_open: boolean;
+    active_tab: number;
+    is_info_panel_visible: boolean;
     Notifications: React.ComponentType;
 }
 
-const BotNotificationMessages = ({ is_drawer_open, Notifications }: TBotNotificationMessagesProps) => (
+const BotNotificationMessages = ({
+    is_drawer_open,
+    Notifications,
```

Comment:
@sara-fs since `Notifications` are react components sent from the core.ui store, they might kept the name capitalised just for the developers to know that the Notifications is a component. We can rename it with small letters as well, but I'd prefer to keep it like this only... kindly let me know your views 🙇